integration-tests/avro-rpc/pom.xml

```diff
@@ -66,6 +66,11 @@
             <artifactId>camel-quarkus-integration-test-support</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.avro</groupId>
+            <artifactId>avro-ipc-jetty</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
@ppalaga Here is a jetty dependency for tests scope used by tests.

Reply:
I am blind, sorry.